components/dataset/thumbnail.js

```diff
@@ -1,34 +1,50 @@
-import { Box, Image, Link } from 'theme-ui'
 import { Badge } from '@carbonplan/components'
 import { alpha } from '@theme-ui/color'
+import { Box } from 'theme-ui'
 
-export const Thumbnail = ({ thumbnail, name, demo }) => {
-  const fallbackThumbnail = 'https://via.placeholder.com/400x200'
-
+export const Thumbnail = ({ url, color, demo }) => {
   return (
-    <Box mt={3} sx={{ position: 'relative' }}>
-      <Link>
-        <Image
-          src={thumbnail ?? fallbackThumbnail}
-          alt={`Thumbnail for ${name}`}
-          sx={{
-            width: '100%',
-          }}
-        />
-        {demo && (
+    <>
```

Comment:
The nesting of elements and their `position`s don't make sense to me. I think this could be simplified a lot by doing something like

```js
<Box sx={{ height: '200px', position: 'relative' }}>
  <Box
    sx={{
      backgroundImage: `url(${url})`,
      position: 'absolute',
      height: '100%',
      width: '100%',
      filter: 'grayscale(100%)',
      backgroundSize: 'cover',
    }}
  />
  <Box
    sx={{
      backgroundColor: alpha(color, 0.4),
      position: 'absolute',
      height: '100%',
      width: '100%',
      opacity: 0.7,
      mixBlendMode: 'dodge',
    }}
  />

  {demo && (
    <Box
      sx={{
        position: 'absolute',
        top: 0,
        width: '100%',
        height: '100%',
        backgroundColor: alpha('secondary', 0.4),
      }}
    >
      <Badge sx={{ mt: 2, ml: 2 }}>Demo</Badge>
    </Box>
  )}
</Box>
```

Reply:
If you use an `img` tag, you can also easily vertically center the image instead of the always just showing the top of the image with the current `div`-based approach.

```js
<Box
  as='img'
  src={url}
  sx={{
    position: 'absolute',
    height: '100%',
    width: '100%',
    filter: 'grayscale(100%)',
    objectFit: 'cover',
  }}
/>
```